README.md

```diff
@@ -56,3 +57,84 @@ Please see the [Contribution Agreement](CONTRIBUTING.md).
 - [Coral & Transport UDFs: Building Blocks of a Postmodern Data Warehouse](https://www.slideshare.net/walaa_eldin_moustafa/coral-transport-udfs-building-blocks-of-a-postmodern-data-warehouse-229545076), Tech-talk, Facebook HQ, 2/28/2020.
 - [Transport: Towards Logical Independence Using Translatable Portable UDFs](https://engineering.linkedin.com/blog/2018/11/using-translatable-portable-UDFs), LinkedIn Engineering Blog, 11/14/2018.
 - [Dali Views: Functions as a Service for Big Data](https://engineering.linkedin.com/blog/2017/11/dali-views--functions-as-a-service-for-big-data), LinkedIn Engineering Blog, 11/9/2017.
+
+
+## Coral-as-a-Service
+
+**Coral-as-a-Service** or simply, **Coral Service** is a service that exposes REST APIs that allow users to interact with Coral without necessarily coming from a compute engine. Currently, the service supports an API for query translation between different dialects and another for interacting with a local Hive Metastore to create example databases, tables, and views so they can be referenced in the translation API. The service can be used in two modes: remote Hive Metastore mode, and local Hive Metastore mode. The remote mode uses an existing (already deployed) Hive Metastore to resolve tables and views, while the local one creates an empty embedded Hive Metastore so users can add their own table and view definitions.
+
+### API Reference
+
+#### /translate
+A **GET** API which takes the following parameters and returns the translated query:
+- `query`: SQL query to translate between two dialects
+- `fromLanguage`: Input dialect (e.g., spark, trino, hive -- see below for supported inputs)
+- `toLanguage`: Output dialect (e.g., spark, trino, hive -- see below for supported outputs)
+
+#### /create
+A **POST** API which takes a SQL query to create a database/table/view in the local metastore
+(note: this endpoint is only available with Coral Service in local metastore mode).
+
+### Instructions to use with examples
+1. Clone [Coral repo](https://github.com/linkedin/coral)
+```bash  
+git clone https://github.com/linkedin/coral.git  
+```  
+2. From the root directory of Coral, access the coral-service module
+```bash  
+cd coral-service  
+```  
+3. Build
+```bash  
+../gradlew clean build  
+```  
+#### To run Coral Service using the **local metastore**:
+4. Run
+```bash  
+../gradlew bootRun --args='--spring.profiles.active=localMetastore'  
+```  
+Example workflow using local metastore:
+
+(Note: Use an [online URL encoder](https://www.urlencoder.org/) to encode SQL queries.)
+
+5. Create a database called `db1` in local metastore using the /create endpoint
+```
+createQuery (non URL encoded): CREATE DATABASE IF NOT EXISTS db1
```

Comment:
We should refer to the parameter here as `statement` instead. Could you update the parameter name and all references in the text?